opencost/opencost
# Add support in CloudCosts for AmortizedCost field

## What does this PR change?
* Adds support for AmortizedCost field in CloudCosts

## Does this PR relate to any other PRs?
* Required for https://github.com/kubecost/kubecost-cost-model/pull/1489

## How will this PR impact users?
* AmortizedCost is desirable on AWS, and now that will be possible.

## Does this PR address any GitHub or Zendesk issues?
* Closes https://kubecost.atlassian.net/browse/CORE-254

## How was this PR tested?
* Inspect query manually
* Use API manually (cannot check accuracy of figures)
* Unit tests

## Does this PR require changes to documentation?
* Not sure
